custom_components/bond/fan.py

```diff
@@ -5,27 +5,25 @@
     SPEED_LOW,
     SPEED_MEDIUM,
     SPEED_HIGH,
-    FanEntity
+    FanEntity,
 )
 
-from bond import (
```

Comment:
Can you please revert all formatting based changes.

Reply:
All set